packages/matter.js/src/behavior/system/commissioning/CommissioningBehavior.ts

```diff
@@ -235,6 +260,7 @@ export namespace CommissioningBehavior {
 
     export class State implements CommissioningOptions {
         commissioned = false;
+        commissionedFabrics: Record<FabricIndex, ExposedFabricInformation> = {};
```

Comment:
Is there an "uncommissionedFabric"?  Wondering if we should just call this "fabrics" since it seems to map 1:1 with known fabrics

Reply:
Valid. I also thought about it. Should I then also rename the commissionedFabricsChanged event to just be fabricsChanged? (And before you ask I would leave the event because it transports info about the change itself (fabricindex and action). The normale state object change event can not transport this.